shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -345,6 +345,7 @@ public SplashScreen provideSplashScreen() {
    * manifest file.
    */
   @Nullable
+  @SuppressWarnings("deprecation")
```

Comment:
Is this needed after the changes to the method?

Reply:
This is needed because of the call to Resources.getDrawable without a theme on older platforms

Reply:
oh, ok, sounds good